LQResultProducer.py

```diff
@@ -8867,11 +8901,16 @@ def ShapeSystematic(channel,normalWeight,presel,cutFile):
 				Z  = QuickIntegral(t_ZJets,selection[0]+'*'+str(Rz_uujj),1.0)
 				W  = QuickIntegral(t_WJets,selection[0],1.0)
 				tt = QuickIntegral(t_TTBar,selection[0]+'*'+str(Rtt_uujj),1.0)
-				VV = QuickIntegral(t_DiBoson,selection[0],1.0)
+				VV = QuickIntegral(t_DiBoson,selection[0]+'*'+str(Rvv_uujj),1.0)
+				TTV = QuickIntegral(t_TTV,selection[0]+'*'+str(Rvv_uujj),1.0)
+				VV += TTV
```

Comment:
QuickIntegral returns [val,err]. You have to combine the separately, using error propagation for the error

VV[0]+=TTV[0]
tmp=sqrt(VV[1]*VV[1] + TTV[1]*TTV[1])
VV[1]=tmp

Reply:
Good catch. 

"I think TTV is small enough we don't need a shape systematic. We needed to include it in the SF to get high enough purity in the control region."

Do we still want to include ttV in the VV shape systematic, in that case? Do we want to keep the VV shape systematic pure VV?

Reply:
Oh I see, you mean perform the shape systematic after you already have added TTV in? If that is the case, yes I agree go ahead and do that.